### PR TITLE
PLANET-7209: Move Reading Time option to Reading settings

### PR DESCRIPTION
**Pull request by lithrel**
Ref: https://jira.greenpeace.org/browse/PLANET-7209

Screenshot from 2023-08-07 11-22-33

## Test

The Reading time WPM option should now be in `Settings > Reading`.

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [atlas](https://www-dev.greenpeace.org/test-atlas) | [admin](https://www-dev.greenpeace.org/test-atlas/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-atlas/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-atlas) | [composer-local.json](https://github.com/greenpeace/planet4-test-atlas/blob/main/composer-local.json)

:watch: 2023.08.04 09:48:10

File: src/Post.php [hunk 1]
```diff
@@ -554,9 +553,9 @@ public function reading_time(): ?int
         }
 
         $locale = $this->get_locale();
-        $wpm = planet4_get_option('reading_time_wpm', Post\ReadingTimeCalculator::DEFAULT_WPM);
+        $wpm = Settings\ReadingTime::get_option();
         $rt = new Post\ReadingTimeCalculator($locale, $wpm);
-        $time = (int) max(1, round($rt->get_time($this->content) / 60));
+        $time = (int) max(1, round($rt->get_time($this->content() ?? '') / 60));
```
Comment (mleray) [inline src/Post.php:558]: Shouldn't we use `0` here instead of `''`?

Comment (lithrel) [reply src/Post.php:558]: ah, no it's the post content, I want it to be a string even if content() returns null or fails - ReadingTimeCalculator::get_time() takes a string as arg

Comment (mleray) [reply src/Post.php:558]: Ah, right, I got confused with the parenthesis 🙈 makes sense!

**Pull request merged**